.mergify.yml

```
@@ -12,7 +12,7 @@ pull_request_rules:
   - name: Notify conflict
     conditions:
       - conflict
-      - closed=false
+      - -closed
```

Thanks for the catch! Maybe this works?
```suggestion
    conditions:
      - and:
        - conflict
        - closed=false

---

oh... let me try it!